src/picongpu/include/initialization/InitialiserController.hpp

```diff
@@ -87,6 +87,25 @@ class InitialiserController : public IInitPlugin
         log<picLog::SIMULATION_STATE > ("Loading from persistent data finished");
     }
 
+    /** log omega_pe for any species
+     *
+     * calculate omega_pe for any given species and create a `picLog::PHYSICS`
+     * log message
+     */
+    template<typename T_Species=bmpl::_1>
+    struct LogOmegaPe
+    {
+        void operator()()
+        {
+            typedef typename T_Species::FrameType FrameType;
+            const float charge=frame::getCharge<FrameType>();
+            const float mass=frame::getMass<FrameType>();
+            log<picLog::PHYSICS >("species %2%: omega_pe * dt <= 0.1 ? %1%") %
```

Comment:
only `omega_p` or `omega_p+getName()`

the `e` in `omega_pe` was for electrons